src/services/assetService.ts

```diff
@@ -40,7 +40,12 @@ export class AssetService {
             filePath = encodeFileURI(filePath, true);
         }
 
-        const md5Hash = new MD5().update(filePath).digest("hex");
+        const hashTarget = JSON.stringify({
```

Comment:
This is just one way to create the string. I wasn't sure if we'd ever need to "go the other way" (reverse the hash and parse out the project file & file path). We can also just concatenate the strings. Open to feedback